components/server/src/server.rs

```diff
@@ -1292,20 +1292,17 @@ impl<ER: RaftEngine> TiKvServer<ER> {
         };
 
         // Determine the base cpu quota
-        let base_cpu_quota = {
+        let base_cpu_quota =
```

Comment:
Very confused about these 1000_f64. L1298 is multiplied, but L1304 is divided.

Reply:
@BornChanger Help us out here

Reply:
> @BornChanger Help us out here

quota_limiter.cputime_limiter() gives the value in million, while quota for set_cpu_time() accepts value in thousands.

Reply:
i.e., to express 1 vcpu, quota_limiter.cputime_limiter() is 1,000,000, set_cpu_time() needs the parameter of 1,000. It's the reality currently and makes code ugly.